gossip/src/cluster_info.rs

```diff
@@ -2880,62 +2904,99 @@ impl Node {
         let (gossip_port, (gossip, ip_echo)) =
             Self::get_gossip_port(&gossip_addr, port_range, bind_ip_addr);
 
-        let (tvu_port, tvu_sockets) =
-            multi_bind_in_range(bind_ip_addr, port_range, num_tvu_sockets.get())
-                .expect("tvu multi_bind");
-        let (tvu_quic_port, tvu_quic) = Self::bind(bind_ip_addr, port_range);
+        let tvu_config = SocketConfig::default().reuseport(true);
```

Comment:
nit: For consistency with higher in this function, it seems we might name this `tvu_udp_config`. There are a couple cases below too; I don't feel too strongly which way we go but I do prefer consistency

Reply:
so was thinking for this PR, since we are just adding this default, we just stick with:
```
let socket_config = SocketConfig::default();
// and
let socket_config_reuseport = SocketConfig::default().reuseport(true)
```

for the next PR when we change buffer sizes, maybe it makes sense to label these explicitly like `tvu_config`, `tpu_forwards_config`, etc. and here we would be setting buffer sizes.